extensions/flow/changelog.md

```diff
@@ -1,5 +1,10 @@
 # Flow Changelog
 
+## Start Commands & Keywords (2022-03-16)
+- Improve command search by adding keywords like `pause` for command like `Stop Timer`
+- Replace `Next Session` command for `Start Focus Session`
+- Add `Start Next Break` command
+
 ## Categories (2022-03-12)
```

Comment:
```suggestion
## [Start Commands & Keywords] - 2022-03-16
- Improve command search by adding keywords like `pause` for command like `Stop Timer`
- Replace `Next Session` command for `Start Focus Session`
- Add `Start Next Break` command

## [Categories] - 2022-03-12
```
We changed the format of the [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history)